utils/funcexp/func_json_array_insert.cpp

```diff
@@ -0,0 +1,156 @@
+#include "functor_json.h"
+#include "functioncolumn.h"
+#include "constantcolumn.h"
+using namespace execplan;
+
+#include "rowgroup.h"
+using namespace rowgroup;
+
+#include "joblisttypes.h"
+using namespace joblist;
+
+#include "jsonhelpers.h"
+using namespace funcexp::helpers;
+
+namespace funcexp
+{
+CalpontSystemCatalog::ColType Func_json_array_insert::operationType(FunctionParm& fp,
+                                                                    CalpontSystemCatalog::ColType& resultType)
+{
+  return fp[0]->data()->resultType();
+}
+
+string Func_json_array_insert::getStrVal(rowgroup::Row& row, FunctionParm& fp, bool& isNull,
+                                         execplan::CalpontSystemCatalog::ColType& type)
+{
+  const string_view jsExp = fp[0]->data()->getStrVal(row, isNull);
+  if (isNull)
+    return "";
+
+  const CHARSET_INFO* cs = fp[0]->data()->resultType().getCharset();
```

Comment:
Some part of this function is the same, as function above, I would like to suggest to not copy past it, but just implement it once and use everywhere you need.